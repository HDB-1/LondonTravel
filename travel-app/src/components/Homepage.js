import React from 'react'

export default function Homepage() {
    return (
        <div>
            <h1>Welcome to the London Travel App</h1>
        </div>
    )
}
