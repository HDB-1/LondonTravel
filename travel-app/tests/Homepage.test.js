import React from 'react';
import { shallow } from 'enzyme';
import Homepage from '../src/components/Homepage';

describe("shallow Homepage", () => {
    let wrapper;

    beforeEach(() => {
        wrapper = shallow(<Homepage />)
    })
    it("should render correctly", () => {
        expect(wrapper).toMatchSnapshot();
    })
    it("should contain an h1 tag", () => {
        expect(wrapper.find("h1").length).toEqual(1)
    })
})