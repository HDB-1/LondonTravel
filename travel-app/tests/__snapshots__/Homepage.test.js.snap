// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shallow Homepage should render correctly 1`] = `
<div>
  <h1>
    Welcome to the London Travel App
  </h1>
</div>
`;
